package.json

```diff
@@ -3,7 +3,7 @@
   "publishConfig": {
     "registry": "https://npm.pkg.github.com/alex-coolcats"
   },
-  "version": "0.1.84",
+  "version": "0.1.85",
```

Comment:
The release process will bump this automatically when this is merged, so just need to change this back to 0.1.84 🙂